app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -53,6 +54,7 @@
     UploadMediaDetailsContract.View, UploadMediaDetailAdapter.EventListener {
 
     private static final int REQUEST_CODE = 1211;
+    public static final String LAST_LOCATION = "last_location_while_uploading";
```

Comment:
It works great! Please add some Javadoc and an example value, though. Thanks!

Reply:
You should also explain that it is a key for applicationKvStore. Place yourself in the shoes of someone who sees this code for the first time and wonders what this String is for. The String itself obviously does not itself contain a location, but that's what the current Javadoc implies. Thanks!